src/object/obj_rpc.h

```diff
@@ -150,6 +150,10 @@ enum obj_rpc_flags {
 	 * oei_epr.epr_hi is epoch.
 	 */
 	ORF_ENUM_WITHOUT_EPR	= (1 << 8),
+	/* CPD RPC leader */
+	DRF_CPD_LEADER		= (1 << 9),
+	/* Bulk data transfer for CPD RPC. */
+	DRF_CPD_BULK		= (1 << 10),
```

Comment:
The bulk is for user's IO data, not CPD RPC request/sub_req right?

Reply:
Yes, it is for user data.